features/restaurant_page.feature

```diff
@@ -22,7 +22,7 @@ Feature: User can see restaurant's details on the restaurant show page
   @googlemap
   Scenario:
```

Comment:
Maybe we should add some content after "Scenario:" while we're at it. :)

Reply:
I'll add it now